boot/zephyr/main.c

```diff
@@ -60,11 +60,12 @@ void main(void)
 	}
 
 	printk("Bootloader chain: 0x%x\n", rsp.br_image_addr);
-	vt = (struct vector_table *)(rsp.br_image_addr +
-				     rsp.br_hdr->ih_hdr_size);
+	vt = (struct vector_table *)(rsp.br_image_addr + TEXT_SECTION_OFFSET);
 	irq_lock();
 	_MspSet(vt->msp);
 
+	printk("Setting vector table to %p\n", vt);
+
 	/* Not all targets set the VTOR, so just set it. */
 	_scs_relocate_vector_table((void *) vt);
```

Comment:
Perhaps this can be removed, now that we're going to be setting this properly in Zephyr.